package.json

```diff
@@ -39,5 +45,10 @@
     "build:docs:api": "bash ./tools/update-api-reference.sh",
     "ci": "lerna run ci --ignore benchmark"
   },
-  "license": "MIT"
+  "lint-staged": {
+    "*.{js,ts,tsx,css}": [
+      "prettier --tab-width 4 --print-width 120 --write",
```

Comment:
Also you can do `--ignore-path .eslintignore` instead of having two files if you'd like.